_pkgdown.yml

```
@@ -96,102 +96,168 @@ news:
     href: https://shiny.posit.co/blog/posts/announcing-new-r-shiny-ui-components/
 
 reference:
-- title: Create a dashboard
-  contents:
+- title: Page Layouts
+  desc: >
+    Page layouts are the basic building blocks of any UI. bslib pages always
+    include Bootstrap and can be themed with bslib's [theming capabilities](#theming).
+- subtitle: Dashboard layouts
+  desc: >
+    These page layouts work best for dashboards. For a single-page dashboards
+    with an optional sidebar, try `page_sidebar()`. For multi-tab dashboards
+    organized as pages in a navbar, also with an optional sidebar, try
+    `page_navbar()`. For a full-page dashboard with items that fill the screen,
+    try `page_fillable()`.
+- contents:
     - page_sidebar
-- title: Page layouts
-  contents:
+    - page_navbar
+    - page_fillable
+- subtitle: Basic page layouts
+  desc: >
+    bslib variants of classic Shiny page layouts.
+- contents:
     - page
-- title: Theme creation
-  description: |
-    Tools for creating customized Bootstrap themes.
+
+- title: User Interface Layouts
+  desc: >
+    These functions help you layout the user interface (UI) elements of your
+    app.
+- subtitle: Multiple columns
+  desc: |
+    Organize inputs and outputs into Bootstrap's
```

This distinction seems important since most of our components are neither input nor output

```suggestion
    Organize UI elements into Bootstrap's
```